2021-10-13-github-templates.md

```diff
@@ -0,0 +1,123 @@
+# OmbuLabs Standard Github Templates
+
+## Problem:
+
+New and junior contributors might be unsure about the information we require to
+efficiently review a PR. Similarly, it might be unclear what information is required when
+a new BUG or FEATURE is logged as an issue in Github.
+
+We have existing PR and ISSUE templates but they are inconsistent and might ask for
+unnecessary information from the contributor.
+
+The goal is to simplify  and streamline the steps required to make a contribution or log
+an issue.
+
+## Proposal:
+
+Come up with a consistent set of Github templates. The templates should include prompts
+only the ESSENTIAL information we require. We want to make sure that we don't ask for more
+information than we require - as this will discourages contributions.
+
+## Templates:
+
+### Pull Requests
+
+```
+    <!--- Before we start... -->
+    <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+    <!--- Did you update any relevant documentation -->
+    <!--- Did you add tests to cover new changes -->
+
+    ## Description
+    <!--- Describe your changes in detail -->
+    <!--- Please read the README before submitting pull requests for this project. -->
+
+    ## Motivation and Context
+    <!--- Why is this change required? What problem does it solve? -->
+    <!--- If it fixes an open issue, please link to the issue here. -->
+
+    ## How Has This Been Tested?
+    <!--- Include any relevant details about your testing environment and the steps you followed -->
+
+    ## Screenshots (if appropriate):
+
+    **I will abide by the [code of conduct](<LINK TO CODE OF CONDUCT HERE>)**
+```
+
+
+### Issues: Bug Reports
+
+```
+    <!--- Before we start... -->
+    <!--- Did you check the documentation for answers? -->
+    <!--- Did you make sure that this bug has not already been reported? -->
+
+    ## Expected Behavior
+    <!--- Tell us what should happen -->
+
+    ## Actual Behavior
+    <!--- Tell us what happens instead -->
+
+    ## Possible Fix
+    <!--- Optionally suggest a fix or reason for the bug -->
+
+    ## Steps to Reproduce
+    <!--- Provide a link to a live example, or an unambiguous set of steps to -->
+    <!--- reproduce this bug. Include code to reproduce, if relevant -->
+    1.
+    2.
+    3.
+    4.
+
+    ## Additional Information
+    <!--- Include any relevant details about your environment (branch, browser, OS) -->
```

Comment:
Hmm I think these are good ideas to have but just if they came up often. I don't think that is the case in our projects atm. When was the last time it was vital to know the Branch, Browser or OS to fix something in our OSS projects?

Reply:
The idea is that it's not required but just some examples of info that somebody can put in this section, maybe branch or OS is not that useful, but knowing which browser or if it's mobile/desktop is useful. But that's just an example of what poeple can share when reporting an bug if it seems relevant.

Reply:
so this will be adapted to each project? so some projects would need it and some not?